gammapy/catalog/hess.py

```diff
@@ -62,6 +63,12 @@ def __str__(self):
         )
         return ss
 
+    def _repr_html_(self):
```

Comment:
A naive question: why should we repeat it these lines when it is implemented in  the mother class `SourceCatalogObject`?